modules/pytket-ionq/pytket/extensions/ionq/backends/ionq.py

```diff
@@ -135,7 +145,7 @@ def available_devices(cls, **kwargs: Any) -> List[BackendInfo]:
                 cls.__name__,
                 "qpu",
                 __extension_version__,
-                IONQ_N_QUBITS,
+                11,
```

Comment:
I was trying to avoid requiring an API key for available_devices. 
To get access to the header, I had to remove @classmethod
Please take another look and let me know if the changes are satisfactory

Reply:
OK so I am not sure this is the best way forward for the reason that it is defined as a `@classmethod` in our main `Backend` class [here](https://github.com/CQCL/tket/blob/01d20c13d7a6e180a7dbf9982bbb8557899d0e07/pytket/pytket/backends/backend.py#L421) and we shouldn't deviate from that behaviour in derived classes. There are several solutions to your API key problem, one being to construct the header from key passed as kwarg, the other being to read the key from the config file and subsequently construct the header.